ablySpec/RealtimeClient.connection.swift

```diff
@@ -713,6 +713,56 @@ class RealtimeClientConnection: QuickSpec {
                 }
 
             }
+
+            // RTN12
+            context("close") {
+
+                // RTN12b
+                it("should receive CLOSED action within the default request timeout") {
```

Comment:
As I read RTN12b, what should be tested is that the transport is disconnected and the connection transitions to 'closed' when the close process timeouts. The test looks good but this sentence should describe that.